social-media/sentiment_client.bal

```diff
@@ -0,0 +1,140 @@
+// AUTO-GENERATED FILE. DO NOT MODIFY.
+// This file is auto-generated by the Ballerina OpenAPI tool.
+
+import ballerina/http;
+import ballerina/url;
+
+# This service provides sentiment analysis for text data.
+public isolated client class SentimentClient {
```

Comment:
Can't we use a simple http:Client instead of this?

Reply:
Will be using the `http:Client` first then replace the client with the OpenAPI generated client later. We are going to say about openapi generated clients right? Or should I remove it from the session?

Reply:
Cool, we don't need to explain a lot about it. We can keep the main focus on the http:Client based implementation. Before moving on to the next one, we can just quickly show show it and let them try it out if they are interested. Otherwise, it can be complicated for the most of them.

Reply:
Ack